### DISCUSSION TITLE
Prediction of Vss by different partition coefficient calcuation methods?

### DISCUSSION THREAD

**Discussion by PriKalra** [Open-Systems-Pharmacology/Forum#744] [Questions & Problems] [answered]
Dear all, 
The longer way would be to build different simulations with different calculation methods to analyze the Vss.  Any smarter way to do it?
Thanks!!

**Comment by msevestre** [comment, marked as answer]
@PriKalra Not at the moment, unfortunately. (or at least, I don't see a way that would be faster than cloning the simulation so that they use the same set of building blocks)

**Comment by PriKalra** [comment]
Thank you

It would be nice to have something smarter. Quite a tool for IVIVE and Reverse Dosimetry projects.😊
